tools/udp-test-server/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM rust as builder
+
+RUN apt-get update
+
+RUN apt-get install musl-tools -yq
+
+RUN rustup target add x86_64-unknown-linux-musl
+
+WORKDIR /workspace
+
+COPY Cargo.toml Cargo.toml
+
+COPY Cargo.lock Cargo.lock
+
+COPY src/main.rs src/main.rs
+
+RUN RUSTFLAGS=-Ctarget-feature=+crt-static cargo build --release --target=x86_64-unknown-linux-musl
+
+FROM gcr.io/distroless/static
```

Comment:
small nit :) but any reason not to just use `scratch` here?

Reply:
No specific reason not to.